pack-translator.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+var fs = require("fs");
+var path = require("path");
+var program = require('commander');
+
+var LocalPackageSource = require("opent2t/package/LocalPackageSource").LocalPackageSource;
+
+program
+    .version("0.1.0")
+    .usage("<translator name>")
+    .parse(process.argv);
+
+if (program.args.length != 1) {
+    program.outputHelp();
+} else {
+    packTranslator(program.args[0]);
+}
+
+function packTranslator(name) {
```

Comment:
>packTranslator [](start = 9, length = 14)

So with this change, we do not need any package.json validator, since it is being auto-generated. But since we expect users to still tweak (OCF nested schemas), do you think we need validator still ?